Alameen688/mydiary-spa
# #162474625 Improve test coverage

#### What does this PR do?
 - Improves tests coverage
#### Description of Task to be completed?
 - Make test coverage 70%+
#### How should this be manually tested?
 - Pull this branch
 - Run - `npm test`
 - Check the code coverage percentage
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

[#162474625](https://www.pivotaltracker.com/story/show/{162474625})

#### Screenshots (if appropriate)

#### Questions:
